src/pet.js

```diff
@@ -0,0 +1,56 @@
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = 0;
+    this.fitness = 10;
+    this.children = [];
+};
+
+Pet.prototype = {
+    get isAlive() {
+        return this.age < 30 && this.hunger < 10 && this.fitness > 0;
+    }
+  };
+
+Pet.prototype.growUp = function() {
+    if (!this.isAlive) throw new Error("Your pet is no longer alive :(");
+    this.age += 1;
+    this.hunger += 5;
+    this.fitness -= 3;
+};
+
+Pet.prototype.walk = function() {
+    const fitnessIncrementPerWalk = 4;
+    const maximumFitnessLevel = 10;
+    if (!this.isAlive) throw new Error("Your pet is no longer alive :(");
+    this.fitness <= 6 ? this.fitness += fitnessIncrementPerWalk : this.fitness += (maximumFitnessLevel - this.fitness);
+};
+
+Pet.prototype.feed = function() {
+    const hungerDecrementPerFeed = 3;
+    if (!this.isAlive) throw new Error("Your pet is no longer alive :(");
+    this.hunger >= 3 ? this.hunger -= hungerDecrementPerFeed : this.hunger -= this.hunger;
```

Comment:
I would create a constant for minimumHunger, then rewrite this ternary as `this.hunger >= hungerDecrementPerFeed ? this.hunger -= hungerDecrementPerFeed : this.hunger = minimumHunger` If you create the minimumHunger constant on the top, above the constructor, you can use the same constant in this function and in the constructor.